charts/app-config/values-production.yaml

```
@@ -2975,6 +2975,10 @@ govukApplications:
 
   - name: travel-advice-publisher
     helmValues:
+      cronTasks:
+        - name: publish-scheduled-editions
+          task: "publish_scheduled_editions"
+          schedule: "0 0 * * *"
```

Please randomise the minute (so that you're not stacking up load at precisely on the hour) and consider picking an hour that's not midnight.

e.g.
```suggestion
          schedule: "39 2 * * *"
```

(unfortunately there isn't yet a way to express "any time plus/minus x minutes" like there is in some other cron implementations)

---

Thank you @sengi , I have pushed the change and merged 👍